requirements/requirements-examples.txt

```diff
@@ -1 +1,2 @@
-matplotlib
\ No newline at end of file
+matplotlib
+scikit-image
```

Comment:
As mentioned before, we can probably use scipy.misc.ascent() instead of skimage.data.camera() to remove this dependency

Reply:
Great thanks, should now be removed.